pkg/gitreceive/k8s_util.go

```diff
@@ -25,6 +24,11 @@ const (
 	dockerSocketPath = "/var/run/docker.sock"
 )
 
+var (
+	slugBuilderImage   = getenv("SLUGBUILDER_IMAGE_NAME", "quay.io/deisci/slugbuilder:v2-beta")
+	dockerBuilderImage = getenv("DOCKERBUILDER_IMAGE_NAME", "quay.io/deisci/dockerbuilder:v2-beta")
```

Comment:
@kmala can you use the [envconfig](https://github.com/kelseyhightower/envconfig) [struct](https://github.com/deis/builder/blob/master/pkg/gitreceive/config.go#L13-L34) that we have already to get these environment variables?